libs/brig-types/src/Brig/Types/Test/Arbitrary.hs

```diff
@@ -313,13 +313,17 @@ instance Arbitrary UserProfile where
         <*> arbitrary
 
 instance Arbitrary RichField where
-    arbitrary = RichField <$> arbitrary <*> arbitrary
+    arbitrary = RichField <$> arbitrary <*> (arbitrary `suchThat` (/= ""))
 
 instance Arbitrary RichInfo where
     arbitrary = do
-        richInfoFields <- nubOn richFieldType <$> arbitrary
+        richInfoAssocList <- nubOn richFieldType <$> arbitrary
```

Comment:
would be slightly nicer if this nubbing went into a general-purpose smart constructor (even though i don't think we'd use that anywhere else).

Reply:
I thought we are going to delete the assoc lists in future, so it doesn't seem like it is worth the effort.